### ISSUE TITLE
PropTypes has been moved to a separate package

### ISSUE THREAD

**Issue by AndrewJack** [taskrabbit/react-native-parsed-text#33]
React 15.5 and later have moved PropTypes to a separate package
```
Warning: PropTypes has been moved to a separate package. Accessing React.PropTypes is no longer supported and will be removed completely in React 16. Use the prop-types package on npm instead.
```
https://facebook.github.io/react/blog/2017/04/07/react-v15.5.0.html#migrating-from-react.proptypes

Does this library require the outdated version of react native or has an update never been required?

If so can we update it?

**Comment by bleonard** [comment]
Happy to accept that PR. We're not fully up to date and haven't had to do it yet.

**Comment by brunocascio** [comment]
I've just added a PR: https://github.com/taskrabbit/react-native-parsed-text/pull/34

**Comment by AndrewJack** [comment]
Ah thanks!

**Comment by aforty** [comment]
Can we merge #44 and update the package?

**Comment by CaryLandholt** [comment]
react 16 is out and this package breaks due to PropTypes being removed

**Comment by fbartho** [comment]
Fixed in #34
